app/subscriber/src/features/my-reports/edit/content/stories/ContentForm.tsx

```diff
@@ -152,6 +253,26 @@ export const ContentForm: React.FC<IContentFormProps> = ({
             onContentChange?.(values);
           }}
         />
+        <Formik
+          initialValues={initialValues}
+          validationSchema={sentimentFormSchema}
+          onSubmit={() => {}}
+          enableReinitialize={true}
+        >
+          {() => (
+            <Form>
+              <FormikSentiment
+                name="tonePools"
```

Comment:
If you correctly named this component then the Formik `setFieldValue` would work correctly. and you wouldn't need also include the `onSentimentChange` logic.

Reply:
I'd need to check this by chat. I'd love to implement this way :)

Reply:
Not using FormikSentiment component.